docs/store-operations/catalog/graphql-admin/product-images.mdx

```diff
@@ -0,0 +1,510 @@
+
+First, add the image to the product for the global store. You can do so using the [Create a product image](/docs/rest-catalog/products/images#create-a-product-image) endpoint of the REST Management API. 
+
+You can then [add the image to the product](#add-image-to-product) for a channel locale and perform the following: 
+- [Set product image information](#set-product-image-information) for the catalog. You can set global values and overrides for the product's image in a channel locale.
+- [Remove overrides for the product's image in a channel locale](#remove-product-image-overrides-for-a-locale). A channel then inherits global values.
+- [Query product image information](#query-product-image-information), those set at the global level and the overrides.
+
+For a full schema, see the [GraphQL Admin API reference](https://developer.bigcommerce.com/graphql-admin/reference).
+
+## Input fields
+
+Setting or removing information requires that you specify ID fields in the input. For more information on how to specify ID fields, see [Input fields](/docs/store-operations/catalog/msf-international-enhancements#input-fields).
+
+## Add image to product
+
+The following example adds images to a product for a channel locale. You can remove the images using the `removeImagesFromProduct` node.
+
+<Tabs items={['Request', 'Response']}>
+  <Tab>
+
+    ```graphql filename="Example mutation: Add image to product" showLineNumbers copy
+    POST https://api.bigcommerce.com/stores/{{STORE_HASH}}/graphql
+    X-Auth-Token: {{ACCESS_TOKEN}}
+    Content-Type: application/json
+    Accept: application/json
+
+    mutation ($input: AddImagesToProductInput!) {
+      product {
+        addImagesToProduct(input: $input) {
+          images {
+            edges {
+              node {
+                id
+
+                # For the global store         
+                altText   
+                isThumbnail 
+                sortOrder  
+                urlStandard  
+                urlZoom
+
+                # For the channel locale
+                overrides (context: {channelId: "bc/store/channel/1", locale: "en"} ) {
+                  edges {
+                    node {
+                      ... on ProductImagesOverridesForChannelLocale {
+                        context {
+                          channelId
+                          locale
+                        }
+                        altText
+                        sortOrder
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+    ```
+
+    ```json filename="GraphQL variables" showLineNumbers copy
+    {
+      "input": {
+          "productId": "bc/store/product/111",
+          "context": {
+            "channelId": "bc/store/channel/1",
+            "locale": "en"
+          },
+          "ids": ["bc/store/productImage/371", "bc/store/productImage/372"]
+        }
+    }
+    ```
```

Comment:
to add image on global level you need to omit context
```json
    {
      "input": {
          "productId": "bc/store/product/111",
          "ids": ["bc/store/productImage/371", "bc/store/productImage/372"]
        }
    }
```

Reply:
the same for `removeImagesFromProduct`

Reply:
I'm wondering where developers can get the image ID from (i.e. 371 in your example), since they haven't added the product to the image yet (see comment above)

Reply:
Thanks, I've added a sentence next to my examples stating that global level is set by omitting the `context` field